python/svix/api.py

```diff
@@ -125,15 +125,15 @@ def _api(self) -> t.Generator[ApiClass, None, None]:
 class Authentication(ApiBase[AuthenticationApi]):
     _ApiClass = AuthenticationApi
 
-    def dashboard_access(self, app_id: str) -> DashboardAccessOut:
+    def dashboard_access(self, app_id: str, idempotency_key: str = "") -> DashboardAccessOut:
```

Comment:
```suggestion
    def dashboard_access(self, app_id: str, idempotency_key: t.Optional[str] = None) -> DashboardAccessOut:
```
Would setting default to None work here? If not that's fine but I don't love explicitly setting to "", not sure what issues this could cause down the line, but if you can't set it to `None` by default can you check if the libs now send an empty `Idempotency-Key` header with your change?